lib/Zonemaster/Backend/DB/MySQL.pm

```diff
@@ -308,6 +309,7 @@ sub add_batch_job {
 		$dbh->do( "CREATE INDEX test_results__params_deterministic_hash ON test_results (params_deterministic_hash)" );
 		$dbh->do( "CREATE INDEX test_results__batch_id_progress ON test_results (batch_id, progress)" );
 		$dbh->do( "CREATE INDEX test_results__progress ON test_results (progress)" );
+		$dbh->do( "CREATE INDEX test_results__domain_undelegated ON test_results (domain, undelegated)" );
```

Comment:
@mtoma, please explain why the index is first dropped before the insert, and then recreated again.

Reply:
This required for maximum performance of INSERTS.
Dropping the indexes improves performance dramatically.

Reply:
@mtoma! What happens to searches that are done just after the insert? Then there is no index and the engine is busy creating the index, isn't it?